src/core/server/logging/logging_system.ts

```diff
@@ -167,17 +167,13 @@ export class LoggingSystem implements LoggerFactory {
   }
 
   private async applyBaseConfig(newBaseConfig: LoggingConfig) {
+    this.enforceBufferAppendersUsage();
+
     const computedConfig = [...this.contextConfigs.values()].reduce(
       (baseConfig, contextConfig) => baseConfig.extend(contextConfig),
       newBaseConfig
     );
 
-    // reconfigure all the loggers without configuration to have them use the buffer
-    // appender while we are awaiting for the appenders to be disposed.
```

Comment:
NIT: should we keep the comments to understand why we are setting the loggers twice?

Reply:
I left it https://github.com/elastic/kibana/pull/98478/files#diff-92a0cd1f9422606d47a5c270e10021d1970ef35ab885406286c0de6e6237ec0cR214
Or you mean something else?

Reply:
right! sorry I missed it :)